app/utilities/drawing/master.py

```diff
@@ -0,0 +1,161 @@
+import sys
+import os
+import PIL
+from PIL import Image, ImageDraw, ImageOps, ImageFont, ImageEnhance, ImageFilter
+from pathlib import Path
+import requests
+from io import BytesIO
```

Comment:
Gonna need @Madscientiste's input on this. Do these count as system imports? If they do, they need to go to the top of the includes with sys and os.